gitpod-io/community-office-hours
# 42/Format-community-highlights-2022

## Description
- Followed the guidelines properly and formatted the 2022 highlights 
- Removed broken links

## Related Issue(s)

Fixes #42 

## How to test
- Go to `community-highlights` folder
- See the changes in `2022.md` file
